pytgcalls/pytgcalls/mtproto/telethon_bridge.py

```diff
@@ -167,7 +167,7 @@ def register_update_handlers(self):
         self.client.add_event_handler(self._process_update, Raw)
 
     async def resolve_and_set_join_as(self, join_as):
-        if self.join_as is None:
+        if join_as is None:
             self.join_as = await self.get_and_set_self_peer()
         else:
             self.join_as = join_as
```

Comment:
Will it work if I pass an username or an id of channel that I own?

Reply:
Yes. Check the join_as parameter here:

https://tl.telethon.dev/methods/phone/join_group_call.html

Reply:
Afaik you can also use "me" and telethon will automatically cast it to InputUserSelf